src/libtoast/src/toast_atm_sim.cpp

```diff
@@ -710,16 +708,20 @@ cholmod_sparse * toast::atm_sim_sqrt_sparse_covariance(
             throw std::runtime_error("cholmod_analyze failed.");
         }
 
-        if (atm_verbose()) {
-            o.str("");
-            o << rank << " : Factorizing sparse covariance ... ";
-            logger.verbose(o.str().c_str());
-        }
+	o.str("");
+	o << rank << " : Factorizing sparse covariance, "
+	  << "nzmax = " << cov->nzmax
+	  << ", (" << cov->nzmax * 8 / pow(2, 20) << "MB)";
+	logger.debug(o.str().c_str());
```

Comment:
Same comment here about removal of `atm_verbose()` optimization.

Reply:
Reverted